tests/init.go

```diff
@@ -29,30 +29,16 @@ var Forks = map[string]*params.ChainConfig{
 	"Frontier": {
 		ChainID: big.NewInt(1),
 	},
-	"EIP150": {
-		ChainID:     big.NewInt(1),
-		EIP150Block: big.NewInt(0),
-	},
-	"EIP158": {
-		ChainID:     big.NewInt(1),
-		EIP150Block: big.NewInt(0),
-		EIP155Block: big.NewInt(0),
-		EIP158Block: big.NewInt(0),
-	},
 	"Byzantium": {
 		ChainID:        big.NewInt(1),
-		EIP150Block:    big.NewInt(0),
```

Comment:
byzantium removal here.

Reply:
I did not remove Byzantium, I removed all references to EIP150